include/player.h

```diff
@@ -0,0 +1,10 @@
+#ifndef PLAYER_H
+#define PLAYER_H
+
+typedef enum {
+    TELEPORT_CHECK_NONE = 0,
+    TELEPORT_CHECK_TO_RTOP = 2,
+    TELEPORT_CHECK_TO_TOP = 4
+} TeleportCheck;
```

Comment:
Everything looks good.

One last nit, if you declare it like this the enum name gets picked up by vscode like this:
![image](https://github.com/Xeeynamo/sotn-decomp/assets/96613413/ba3336f2-dd30-42b5-8713-9128de0805bc)
otherwise it displays it like <unnamed>
![image](https://github.com/Xeeynamo/sotn-decomp/assets/96613413/91edd099-14fa-42f1-9665-cadd41627b5d)
i found this out relatively recently.
```suggestion
typedef enum TeleportCheck {
    TELEPORT_CHECK_NONE = 0,
    TELEPORT_CHECK_TO_RTOP = 2,
    TELEPORT_CHECK_TO_TOP = 4
} TeleportCheck;